src/specs/data_structures.md

```diff
@@ -36,7 +36,7 @@
     - [Transaction](#transaction)
     - [SignedTransactionData](#signedtransactiondata)
       - [SignedTransactionDataTransfer](#signedtransactiondatatransfer)
-      - [SignedTransactionDataPayForMessage](#signedtransactiondatapayformessage)
+      - [SignedTransactionDataMsgPayForData](#signedtransactiondatamsgpayfordata)
```

Comment:
I'm confused by the `SignedTransactionData` prefix here. Is it possible to remove since it doesn't appear to match the current implementation?

[Orthogonal] is it possible to remove all the other `SignedTransactionData*` transactions in this list that aren't unique to Celestia?

Reply:
Yes, in a follow-up PR

Reply:
Created https://github.com/celestiaorg/celestia-app/issues/751